File: java-kotlin/src/test/java/com/likejazz/leetcode/java/solution14/SolutionTest.java
package com.likejazz.leetcode.java.solution14;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SolutionTest {

    @Test
    void longestCommonPrefix() {
        Solution s = new Solution();
        String[] a = {"flower", "flow", "flight"};
        String r = s.longestCommonPrefix(a);

        assertEquals("fl", r);
    }
}

File: java-kotlin/src/test/kotlin/com/likejazz/leetcode/solution1903/SolutionTest.kt
package com.likejazz.leetcode.solution1903

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

internal class SolutionTest {
    private lateinit var s: Solution

    @BeforeEach
    fun setUp() {
        s = Solution()
    }

    @Test
    fun t1() {
        assertEquals("5", s.largestOddNumber("52"))
        assertEquals("", s.largestOddNumber("4206"))
        assertEquals("35427", s.largestOddNumber("35427"))
        assertEquals("35", s.largestOddNumber("3542"))
    }
}